docs/lib/datastore/fragments/flutter/relational/updated-schema.md

```diff
@@ -0,0 +1,26 @@
+For the examples below with DataStore let's add a new model to the [sample schema](~/lib/datastore/getting-started.md#sample-schema):
```

Comment:
add missing comma between DataStore and let's

Reply:
added